src/helpers/folder_helpers.sh

```diff
@@ -6,7 +6,10 @@
 # Changes into the given directory
 function change_directory {
   local directory=$1
-  run_command "cd $directory"
+  local omit_if_doesnt_exist=$2
+  if [ -d "$directory" ] || [ -z "$omit_if_doesnt_exist" ]; then
+    run_command "cd $directory"
+  fi
```

Comment:
Thoughts on just creating a new function for this? `change_directory_if_exists`
That function wraps the checking and then just calls `change_directory`